recipes/r-spp/meta.yaml

```diff
@@ -39,7 +39,7 @@ requirements:
 test:
   commands:
     - $R -e "library('spp')"
-
+    - $R -e "library('spp'); read.bam.tags(system.file('extdata', 'ex1.bam', package='Rsamtools'))"
```

Comment:
can you try "library(\"spp\"); read.bam.tags(system.file(\"extdata\", \"ex1.bam\", package=\"Rsamtools\"))" ... need to play with it